### DISCUSSION TITLE
possibility to rename the claims

### DISCUSSION THREAD

**Discussion by Runred** [GriefPrevention/GriefPrevention#803] [Ideas]
**As** a player In a (90%) vanilla server
It would be cool to add the option of being allowed to rename the claims I own
Because I have way too many and It's very hard to find them with the /claimlist command
3 commands would be added:  
`/nameclaim`  `/renameclaim`  `/deleteclaimname`

The result would be like this:
*when in a claim a player wants to rename* `/nameclaim "mob_farm"`
*when a player wants to set a new name* `/nameclaim "mob_farm"` or `/renameclaim "Spawner"`
*when a player wants to delete the name* `/deletenameclaim` or `/delnameclaim` or `/dnc` (this one is very optionnal)

The claims should appear like this with the `/claimlist` command 

***1600 blocks available + 0 bonus = 1600 blocks in total
Overworld: **Spawner**  x1100 z-1100 (-100 blocks)
Flatworld: **Mob_Farm_Prototype** x100 z-100 (-1000 blocks)
= 500 blocks left  to use ***

Adding this feature would be very interesting for me and for all the members of my server which is meant to be the most vanilla possible with few plugins.

I hope you will respond to me even if you can't do it or if they are problems ^^
(sorry if i didn't use the template I was confused im new on GH)

**Comment by Jikoo** [comment]
Sounds like you're after [ClaimClassifier](https://dev.bukkit.org/projects/claimclassifier).
